sonar-application/src/main/java/org/sonar/application/JdbcSettings.java

```diff
@@ -91,6 +119,27 @@ void checkUrlParameters(Provider provider, String url) {
     }
   }
 
+  private static void warnIfUrlIsSet(String port, String existing, String expectedUrl) {
+    if (isNotEmpty(existing)) {
+      Logger logger = LoggerFactory.getLogger(JdbcSettings.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1312)